packages/network-controller/src/NetworkController.ts

```diff
@@ -56,7 +57,7 @@ const log = createModuleLogger(projectLogger, 'NetworkController');
 export type ProviderConfig = {
   rpcUrl?: string;
   type: NetworkType;
-  chainId: Hex;
+  chainId: string;
```

Comment:
I think we should just rename all of these to `caipChainId`. Can be pretty confusing what `chainId` is in different contexts